### ISSUE TITLE
sinfo : command not found, why ?

### ISSUE THREAD

**Issue by nyue** [aws-samples/pcluster-manager#99]
I just spurn up a cluster and chose slurm

I ssh to the head node to run sinfo to check on the cluster

It says sinfo not found.

I just created a different cluster about 12 hours ago and a couple of days ago and sinfo is installed.

Not sure if this is a regression or something else.

Cheers

**Comment by nyue** [comment]
My bad. It is there in /opt/slurm/bin

I guess that path is no longer in $PATH

Maybe it should.

**Comment by sean-smith** [comment]
Can you provide your cluster config? Any modifications to `~/.bashrc`, any Custom bootstrap scripts?

It should be on the path and is on the latest clusters I've spun up.

**Comment by sean-smith** [comment]
Is this on Ubuntu? https://github.com/aws-samples/pcluster-manager/issues/96

**Comment by nyue** [comment]
yes it is on Ubuntu but I am connecting via NICE DCV so it has proper terminal shell right ?

```
HeadNode:
  InstanceType: t2.small
  Ssh:
    KeyName: testssh
  Networking:
    SubnetId: subnet-ec46f185
  Iam:
    AdditionalIamPolicies:
      - Policy: arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
  Dcv:
    Enabled: true
Scheduling:
  Scheduler: slurm
  SlurmQueues:
    - Name: queue0
      ComputeResources:
        - Name: queue0-t2medium
          MinCount: 2
          MaxCount: 2
          InstanceType: t2.medium
      Networking:
        SubnetIds:
          - subnet-ec46f185
Region: ca-central-1
Image:
  Os: ubuntu2004
SharedStorage:
  - Name: Ebs0
    StorageType: Ebs
    MountDir: /shared
    EbsSettings:
      VolumeType: gp2
      DeletionPolicy: Delete
      Size: 35
```

**Comment by sean-smith** [comment]
Ya should be correct via DCV

**Comment by sean-smith** [comment]
I opened an issue with the ParallelCluster team on the DCV portion of this. 

For the SSM version, the reason it's not working is a difference between `sh` and `bash`, `bash` has the source command but sh does not. https://askubuntu.com/questions/504546/error-message-source-not-found-when-running-a-script

We'll fix this, DCV will require a patch in pcluster.

**Comment by charlesg3** [comment]
https://github.com/aws/aws-parallelcluster-cookbook/pull/1417

**Comment by sean-smith** [comment]
Work around until then:

```bash
sudo su - ubuntu
```

**Comment by sean-smith** [comment]
Resolved in `3.1.3` to upgrade launch a new version of pcluster-manager.

See https://github.com/aws/aws-parallelcluster/issues/4004
